package.json

```diff
@@ -1060,7 +1060,7 @@
         "@microsoft/eslint-config-azuretools": "^0.1.0",
         "@microsoft/vscode-azext-dev": "^0.1.4",
         "@types/fs-extra": "^8.1.0",
-        "@types/gulp": "^4.0.6",
```

Comment:
Just curious, why was this needed?

Reply:
I think I ran an npm audit it just auto updated it. There were security vulnerabilities that I didn't quite understand.